pom.xml

```diff
@@ -416,6 +418,8 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <configuration>
+            <source>1.7</source>
+            <target>1.7</target>
```

Comment:
I believe it would be better to override the source/target properties coming from parent POM(s) (those are then used by the compiler plugin). Specifically `<maven.compiler.target>1.7</maven.compiler.target>` and `<maven.compiler.source>1.7</maven.compiler.source>`. See https://github.com/jboss-integration/jboss-integration-platform-bom/blob/6.0.x/pom.xml#L94 for more info.